tst/regression/test_suites/advection_convergence/advection_convergence.py

```diff
@@ -73,6 +76,9 @@ def Prepare(self,parameters, step):
                 ]
         # TEST: Advection only in y-direction
         elif step <= 2*n_res:
+            # Only run coverage for 32 case
+            if lin_res[step % n_res - 1] == 32:
```

Comment:
I'd add a similar check to to enable test coverage for one case in the z-direction (i.e, where nx3 > 1) below as it's the one case not covered yet (3D, no AMR, single MeshBlock per Mesh).